Embedded/MaxMix/Display.ino

```diff
@@ -88,23 +89,30 @@ void DisplayGameSelectScreen(Adafruit_SSD1306* display, char* name, uint8_t volu
 //---------------------------------------------------------
 // Draws the Application mode Edit screen
 //---------------------------------------------------------
-void DisplayApplicationEditScreen(Adafruit_SSD1306* display, char* name, uint8_t volume, bool isMuted, uint8_t modeIndex, uint8_t modeCount)
+void DisplayApplicationEditScreen(Adafruit_SSD1306* display, char* name, uint8_t volume, bool isMuted, uint8_t modeIndex, uint8_t modeCount, uint32_t* displayScrollTimer, uint32_t now)
 {
   display->clearDisplay();
-  
-  DrawDotGroup(display, modeIndex, modeCount);
 
   uint8_t x0, y0, x1, y1;
+  uint16_t scrollOffset = ScrollOffset(displayScrollTimer, now, strlen(name), 0, DISPLAY_CHAR_WIDTH_X1, DISPLAY_CHAR_SPACING_X1, DISPLAY_CHAR_MAX_WIDTH_X1, DISPLAY_SCROLL_STEP_INTERVAL_X1);
+
+  if (strlen(name) <= DISPLAY_CHAR_MAX_X1)
+    scrollOffset = 0;
 
-  // Item name
   display->setTextSize(1);
   display->setTextColor(WHITE);
-  display->setCursor(DISPLAY_AREA_CENTER_MARGIN_SIDE, 0);
+  display->setCursor(DISPLAY_AREA_CENTER_MARGIN_SIDE - scrollOffset, 0);
 
-  x0 = min(DISPLAY_CHAR_MAX_X1, strlen(name)); // Name length
-  for(size_t i = 0; i < x0; i++)
+  uint8_t len = min(strlen(name), DISPLAY_CHAR_MAX_X1 + 1 + (scrollOffset / (DISPLAY_CHAR_WIDTH_X1 + DISPLAY_CHAR_SPACING_X1)));
+  for (size_t i = 0; i < len; i++)
     display->print(name[i]);
 
+  // Mask texts
+  display->fillRect(0, 0, DISPLAY_AREA_CENTER_MARGIN_SIDE, DISPLAY_CHAR_HEIGHT_X1, 0);
+  display->fillRect(DISPLAY_WIDTH - DISPLAY_AREA_CENTER_MARGIN_SIDE + 1, 0, DISPLAY_WIDTH, DISPLAY_CHAR_HEIGHT_X1, 0);
```

Comment:
The +1 here is somewhat a "magic number" but I'm not sure if it makes sense to define a const just for this. 
The +1 here is caused by the side margin (DISPLAY_AREA_CENTER_MARGINE_SIDE = 11) max char width (DISPLAY_CHAR_MAX_WIDTH_X1 = 107) not adding up to the display width (128). 

The way I thought it should work would be DISPLAY_WIDTH = 2 (MARGIN_SIDE) + CHAR_MAX_WIDTH
But currently, if you calculate this you'll get 128 = 129, and thus the +1 to offset the mask.

Thoughts?